requirements.txt

```diff
@@ -1,6 +1,10 @@
 inputs==0.1
 wxPython==4.0.0a1
-matplotlib==1.5.2
-numpy==1.10.1
-scikit-image==0.12.3
+matplotlib==2.0.0
+numpy==1.12.1
+scikit-image==0.13.0
```

Comment:
No they are not required, I updated my virtualenv back to your requirements. I mostly try to keep them up-to date that is the only reason.